From a68d863151dead2ce6d54a8838607a771f78c4b0 Mon Sep 17 00:00:00 2001
Message-Id: <a68d863151dead2ce6d54a8838607a771f78c4b0.1387369730.git.minovotn@redhat.com>
In-Reply-To: <091eecc4fa42754760dfff393dabcc2b444e9693.1387369730.git.minovotn@redhat.com>
References: <091eecc4fa42754760dfff393dabcc2b444e9693.1387369730.git.minovotn@redhat.com>
From: Markus Armbruster <armbru@redhat.com>
Date: Tue, 10 Dec 2013 15:29:10 +0100
Subject: [PATCH 10/21] qapi: lack of two commas in dict

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1386689361-30281-8-git-send-email-armbru@redhat.com>
Patchwork-id: 56136
O-Subject: [PATCH 7.0 qemu-kvm 07/18] qapi: lack of two commas in dict
Bugzilla: 997915
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>
RH-Acked-by: Luiz Capitulino <lcapitulino@redhat.com>

From: Amos Kong <akong@redhat.com>

Signed-off-by: Amos Kong <akong@redhat.com>
Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>
(cherry picked from commit ad0f171e25fb60b385dc7d07632916d3b1043d81)
Signed-off-by: Markus Armbruster <armbru@redhat.com>
---
 qapi-schema.json | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qapi-schema.json | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/qapi-schema.json b/qapi-schema.json
index 92fcd54..b27108f 100644
--- a/qapi-schema.json
+++ b/qapi-schema.json
@@ -447,7 +447,7 @@
 #
 # Since: 1.4
 ##
-{ 'enum': 'DataFormat'
+{ 'enum': 'DataFormat',
   'data': [ 'utf8', 'base64' ] }
 
 ##
@@ -1693,7 +1693,7 @@
 #
 # Since: 1.1
 ##
-{ 'enum': 'NewImageMode'
+{ 'enum': 'NewImageMode',
   'data': [ 'existing', 'absolute-paths' ] }
 
 ##
-- 
1.7.11.7

